tests/includes/API/ApiExceptionTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
This file was also added by you in be87a2f - you may want to rebase onto the most recent newinternal commit and resolve the conflicts.